src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static final Integer VALID_VALUE = 10;

```

Comment:
```suggestion
    private static final Integer DEFAULT_AMOUNT = 10;
```
the same for other test classes for handlers